### PR TITLE
Add korean channels

### PR DESCRIPTION
**Pull request by allanf181**

### CODE REVIEW

**Review by BellezaEmporium — DISMISSED** [review]
I think you just unveiled a new pair of possibilities.

File: streams/kr.m3u [hunk 1]
```diff
@@ -53,30 +55,78 @@ http://mediaworks.dema.mil.kr:1935/live_edge/cudo.sdp/playlist.m3u8
...
+#EXTINF:-1 tvg-id="SPOTV2.kr" status="online",SPOTV2 (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/spotv_2_540/playlist.m3u8
+#EXTINF:-1 tvg-id="TVChosun.kr" status="online",TV Chosun (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/tvchosun_540/playlist.m3u8
 #EXTINF:-1 tvg-id="TBSTV.kr" status="online",TBS Seoul (720p)
 https://cdntv.tbs.seoul.kr/tbs/tbs_tv_web.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Tooniverse.kr" status="online",Tooniverse (576p) [Not 24/7]
+http://51.81.66.195:1935/krtv/tooniverse_540/playlist.m3u8
+#EXTINF:-1 tvg-id="tvN.kr" status="online",tvN (576p) [Not 24/7]
+http://51.81.66.195:1935/krtv/tvn_540/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/kr.m3u:121]: DMCAd

**Comment by LaneSh4d0w** [comment]
IP DNS tells me it's cdn.us195.jpnettv.live

**Comment by LaneSh4d0w** [comment]
Japanese Animax : http://cdn.us195.jpnettv.live:1935/jptv/janimax_540/playlist.m3u8

**Comment by allanf181** [comment]
they have several domains, the channels I found were on cdn.th136.jpnettv.live

**Comment by LaneSh4d0w** [comment]
> they have several domains, the channels I found were on cdn.th136.jpnettv.live

Interesting, well, you now have 2 pairs of channels :)

**Comment by allanf181** [comment]
https://subdomainfinder.c99.nl/scans/2021-12-17/jpnettv.live

File: streams/kr.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply streams/kr.m3u:123]: TvN

Comment (allanf181) [reply streams/kr.m3u:123]: https://github.com/iptv-org/iptv/issues/1831 this?

Comment (allanf181) [reply streams/kr.m3u:123]: the tvn I added is not from Nagravision, it's from CJ E&M

Comment (allanf181) [reply streams/kr.m3u:123]: https://en.wikipedia.org/wiki/TvN_(Asian_TV_channel)

**Comment by allanf181** [comment]
Sky Sports is not from DFL Deutsche Fussball Liga e.V,  it's from https://en.wikipedia.org/wiki/KT_Corporation
https://en.wikipedia.org/wiki/Sky_Sports_(South_Korean_TV_channel)

File: streams/kr.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply streams/kr.m3u:123]: Well they have the rights to "encrypt" their stream. While the channel is from CJ E&M, Nagravision is the society that provides encryption for those channels, in partnership with them.

**Comment by LaneSh4d0w** [comment]
> Sky Sports is not from DFL Deutsche Fussball Liga e.V, it's from https://en.wikipedia.org/wiki/KT_Corporation https://en.wikipedia.org/wiki/Sky_Sports_(South_Korean_TV_channel)

It is not the channel that is an issue there, it's the content of their streams. Sky Sports did air some Bundesliga, which is what the DFL has the rights.

**Comment by allanf181** [comment]
http://sky-k.co.kr according to the website they only broadcast korean football

**Comment by allanf181** [comment]
https://www.dfl.de/en/news/cj-entertainment-and-media-becomes-home-of-the-bundesliga-in-korea-after-three-year-deal-announced/

File: streams/kr.m3u (hunk 1 above)
Comment (allanf181) [reply streams/kr.m3u:123]: cj e&m never uploaded any dmca to github and i didn't find any dmca from companies representing cj e&m

Comment (allanf181) [reply streams/kr.m3u:123]: in this pr there are more CJ E&M channels

**Comment by allanf181** [comment]
https://github.com/iptv-org/database/commit/3450cfc8d2fbc9a0c4a45801bf508ca059188289#diff-34e1620599f4d18fd7d7a1b70a7ae87769b59f3825c7264a06382964e38f7fbbR800

**Comment by allanf181** [comment]
https://github.com/iptv-org/database/pull/100

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**